rpcserver.go

```diff
@@ -5718,6 +5728,17 @@ func handleVerifyMessage(s *rpcServer, cmd interface{}, closeChan <-chan struct{
 	return address.EncodeAddress() == c.Address, nil
 }
 
+// handleVersion implements the version command.
+func handleVersion(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	result := &dcrjson.VersionResult{
+		VersionString: jsonrpcSemverString,
+		Major:         jsonrpcSemverMajor,
+		Minor:         jsonrpcSemverMinor,
+		Patch:         jsonrpcSemverPatch,
+	}
```

Comment:
This doesn't include the `Prerelease` and `BuildMetadata` fields.  I realize they aren't used currently, but they should probably be set here and defined above so the infrastructure is in place.